ksql-engine/src/test/resources/query-validation-tests/aggregate-table.json

```diff
@@ -0,0 +1,34 @@
+{
+  "comments": [
+    "You can specify multiple statements per test case, i.e., to set up the various streams needed",
+    "for joins etc, but currently only the final topology will be verified. This should be enough",
+    "for most tests as we can simulate the outputs from previous stages into the final stage. If we",
+    "take a modular approach to testing we can still verify that it all works correctly, i.e, if we",
+    "verify the output of a select or aggregate is correct, we can use simulated output to feed into",
+    "a join or another aggregate."
+  ],
+  "tests": [
+    {
+      "name": "calculate aggregation on a table",
+      "statements": [
+        "CREATE TABLE TEST (ID bigint, NAME varchar, REGION string) WITH (kafka_topic='test_topic', value_format='DELIMITED', key='ID');",
+        "CREATE TABLE COUNT_BY_REGION AS SELECT REGION, count(*) FROM TEST GROUP BY REGION;"
+      ],
+      "inputs": [
+        {"topic": "test_topic", "key": 0, "value": "0,alice,east", "timestamp": 0},
+        {"topic": "test_topic", "key": 1, "value": "1,bob,east", "timestamp": 0},
+        {"topic": "test_topic", "key": 2, "value": "2,carol,west", "timestamp": 0},
+        {"topic": "test_topic", "key": 3, "value": "3,dave,west", "timestamp": 0},
+        {"topic": "test_topic", "key": 1, "value": "1,alice,west", "timestamp": 0}
```

Comment:
Consider adding a delete / null-value into the mix just for completeness?

Reply:
Thanks for the suggestion here. Looks like we have a general bug with tables where we drop tombstones on the floor. I'll file an issue and fix that separately since we should probably get that in asap.

Reply:
https://github.com/confluentinc/ksql/issues/1169